### ISSUE TITLE
php exec give a status code 127

### ISSUE THREAD

**Issue by GoogleCodeExporter** [chrsmithdemos/minify#240]
```
What steps will reproduce the problem?
1.when i use YUICompressor.jar
2.i get a 500 error

What is the expected output? What do you see instead?

i change the code in YUICompressor.php:

if(!exec(self::_getCmd($options, $type, $tmpFile), $output, $revar)){
            throw new Exception('Minify_YUICompressor command error code: ' . $revar);
};

sorry for my English!

Please use labels and text to provide additional information.

```

---

Original issue reported on code.google.com by `star212...@gmail.com` on 18 Aug 2011 at 9:21

**Comment by GoogleCodeExporter** [comment]
```
php version is 5.1.6
java is 1.6.0
centOS 5
```

---

Original comment by `star212...@gmail.com` on 18 Aug 2011 at 9:26

**Comment by GoogleCodeExporter** [comment]
```
This seems solved in 2.1.5 (exit code is checked)
```

---

Original comment by `elan.ruu...@gmail.com` on 29 Mar 2012 at 7:18

**Comment by GoogleCodeExporter** [comment]
---

Original comment by `mrclay....@gmail.com` on 29 Mar 2012 at 8:04
- Changed state: **Verified**
